js/templates/feedback.html

```diff
@@ -9,7 +9,7 @@
     <form id="feedback-form" class="container">
       <fieldset>
         <legend class="feedback__title">Help us improve betaFEC</legend>
-        <p class="t-sans">Don't include sensitive information like your name, contact information or social security number.</p>
+        <p class="t-sans">Don't include sensitive information like your name, contact information or Social Security number.</p>
```

Comment:
What's our position on the serial comma? Should this be "name, contact, information, or Social Security number"?